CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ Please add to the relevant subsections under Unreleased below on every PR where
 
 - `run.define_metric()` raises an error when given extraneous arguments (@timoffex in https://github.com/wandb/wandb/pull/8040)
 - In disabled mode, use the `wandb.sdk.wandb_run.Run` class instead of `wandb.sdk.wandb_run.RunDisabled`, which has been removed (@dmitryduev in https://github.com/wandb/wandb/pull/8037)
+- Fix process stalling when trying to call `artifact.download()` when `WANDB_MODE = offline` in https://github.com/wandb/wandb/pull/8009
```

Comment:
maybe we should just say changes the behavior to error out when trying to download in offline mode?

Reply:
also you can add yourself as contributor:
- ... by @trane293 in https://github.com/wandb/wandb/pull/8037

this way you will be listed as contributor on the upcoming release :)

Reply:
Done! Thanks for the suggestion and the heads up :)